### PR TITLE
[PT-BR]: translate about resources to pt-br

### PR DESCRIPTION
**Pull request by AugustoCalaca**
ping friendly @khaosdoctor

### CODE REVIEW

**Comment by nschonni** [comment]
Might be better to submit on https://crowdin.com/project/nodejs-website/pt-BR

**Comment by alexandrtovmach** [comment]
@AugustoCalaca Thank you for contribution. We're moving translation process to Crowdin, please read these guidelines for more information:
https://github.com/nodejs/i18n/#contributing

**Comment by alexandrtovmach** [comment]
@nodejs/nodejs-pt Please manage this PR [until **May 1, 2020**](https://github.com/orgs/nodejs/teams/translators/discussions/7)

File: locale/pt-br/about/resources.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: about.hbs
+title: Logos e Gráficos
+---
+
+# Recursos
+
+## Baixar logos
+
+Por favor, revise [política de marca registrada](/pt/about/trademark/) para informações sobre o que é permitido aos usuários das logos e marcas do Node.js®.
```
Comment (JCMais) [inline locale/pt-br/about/resources.md]:
```suggestion
Por favor, revise a [política de marca registrada](/pt/about/trademark/) para informações sobre o uso permitido das logos e marcas do Node.js®.
```

File: locale/pt-br/about/resources.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Orientações para a mostra visual da marca Node.js são descritas em [Orientações Visuais](/static/documents/foundation-visual-guidelines.pdf).
```
Comment (JCMais) [inline locale/pt-br/about/resources.md]:
```suggestion
Orientações para a exibição visual da marca Node.js estão disponíveis em [Orientações Visuais](/static/documents/foundation-visual-guidelines.pdf).
```

File: locale/pt-br/about/resources.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+<table class="logos">
+  <tr>
+    <td class="bg-white"><a href="/static/images/logos/nodejs-new-pantone-black.ai"><img src="/static/images/logos/nodejs-new-pantone-black.svg" alt="Node.js on light background"></a></td>
+    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-pantone-white.ai"><img src="/static/images/logos/nodejs-new-pantone-white.svg" alt="Node.js on dark background"></a></td>
```
Comment (JCMais) [inline locale/pt-br/about/resources.md]:
```suggestion
    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-pantone-white.ai"><img src="/static/images/logos/nodejs-new-pantone-white.svg" alt="Node.js sobre um plano de fundo escuro"></a></td>
```

File: locale/pt-br/about/resources.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Baixar logos
+
+Por favor, revise [política de marca registrada](/pt/about/trademark/) para informações sobre o que é permitido aos usuários das logos e marcas do Node.js®.
+
+Orientações para a mostra visual da marca Node.js são descritas em [Orientações Visuais](/static/documents/foundation-visual-guidelines.pdf).
+
+<table class="logos">
+  <tr>
+    <td class="bg-white"><a href="/static/images/logos/nodejs-new-pantone-black.ai"><img src="/static/images/logos/nodejs-new-pantone-black.svg" alt="Node.js on light background"></a></td>
```
Comment (JCMais) [inline locale/pt-br/about/resources.md]:
```suggestion
    <td class="bg-white"><a href="/static/images/logos/nodejs-new-pantone-black.ai"><img src="/static/images/logos/nodejs-new-pantone-black.svg" alt="Node.js sobre um plano de fundo claro"></a></td>
```

File: locale/pt-br/about/resources.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-pantone-white.ai"><img src="/static/images/logos/nodejs-new-pantone-white.svg" alt="Node.js on dark background"></a></td>
+  </tr>
+  <tr>
+    <td><a href="/static/images/logos/nodejs-new-pantone-black.ai">AI padrão do Node.js</a></td>
+    <td><a href="/static/images/logos/nodejs-new-pantone-white.ai">AI invertida do Node.js</a></td>
+  </tr>
+  <tr>
+    <td class="bg-white"><a href="/static/images/logos/nodejs-new-black.ai"><img src="/static/images/logos/nodejs-new-black.svg" alt="Node.js on light background"></a></td>
```
Comment (JCMais) [inline locale/pt-br/about/resources.md]:
```suggestion
    <td class="bg-white"><a href="/static/images/logos/nodejs-new-black.ai"><img src="/static/images/logos/nodejs-new-black.svg" alt="Node.js sobre um plano de fundo claro"></a></td>
```

File: locale/pt-br/about/resources.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-white.ai"><img src="/static/images/logos/nodejs-new-white.svg" alt="Node.js on dark background"></a></td>
```
Comment (JCMais) [inline locale/pt-br/about/resources.md]:
```suggestion
    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-white.ai"><img src="/static/images/logos/nodejs-new-white.svg" alt="Node.js sobre um plano de fundo escuro"></a></td>
```

**Review by JCMais — CHANGES_REQUESTED** [review]
I've left some suggestions

**Review by JCMais — APPROVED** [review]
looks good, thanks @AugustoCalaca

**Review by TiagoDanin — APPROVED** [review]
:+1: LGTM

**Comment by JCMais** [comment]
@AugustoCalaca could you please rebase this against master?

**Comment by alexandrtovmach** [comment]
@AugustoCalaca Thanks for contribution
@JCMais @TiagoDanin Additional thanks for reviews

**Pull request merged**